libglusterfs/src/inode.c

```diff
@@ -413,8 +414,11 @@ __inode_passivate(inode_t *inode)
     dentry_t *dentry = NULL;
     dentry_t *t = NULL;
 
-    list_move_tail(&inode->list, &inode->table->lru);
-    inode->table->lru_size++;
+    if (!inode->in_lru_list) {
```

Comment:
If this is false, it's a bug that needs to be fixed. Have you found any case where this happens ?